features/plugin-uninstall.feature

```diff
@@ -36,3 +36,13 @@ Feature: Uninstall a WordPress plugin
       Warning: The 'edit-flow' plugin could not be found.
       """
     And the return code should be 0
+
+  Scenario: Uninstall all installed plugin
+    When I run `wp plugin uninstall --all`
+    Then STDOUT should be:
+      """
+      Uninstalled and deleted 'akismet' plugin.
+      Uninstalled and deleted 'hello.php' plugin.
+      Success: Uninstalled 2 of 2 plugins.
+      """
+    And the return code should be 0
```

Comment:
Also should maybe do `When I run the previous command again` step here with STDOUT `Success: No plugins installed.`.

Also add scenario `Uninstall all installed plugins when one or more activated` with one plugin activated and run without and with `--deactivate` flag.

Reply:
@gitlost I am having some problem with this test. Can you please check.